app/managers.py

```diff
@@ -0,0 +1,49 @@
+from models import Actor
+import sqlite3
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self.connect_ = sqlite3.connect("cinema.sqlite")
```

Comment:
Consider using a configuration file or environment variable for the database connection string instead of hardcoding it. This will make your code more flexible and secure.